### PR TITLE
Fixes SiCKRAGETV/sickrage-issues/issues/3039

### PR DESCRIPTION
**Pull request by fernandog**
>>> import sys;sys.path.insert(1,'lib');from sickbeard import common
>>> status = 6404
>>> print common.statusStrings[status]
Downloaded (1080p WEB-DL)
>>> oldStatus, oldQuality = common.Quality.splitCompositeStatus(status)
>>> new_status = common.Quality.compositeStatus(common.ARCHIVED, oldQuality)
>>> print new_status
6406
>>> print common.statusStrings[new_status]
Archived (1080p WEB-DL)

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

**Comment by fernandog** [comment]
@miigotu is this ok?

File: sickbeard/tv.py [hunk 1]
```diff
@@ -1100,8 +1100,16 @@ def refreshDir(self):
...
+                                oldStatus, oldQuality = Quality.splitCompositeStatus(curEp.status)
+                                new_status = Quality.compositeStatus(ARCHIVED, oldQuality)
+                                curEp.status = new_status
+                            else:
+                                curEp.status = sickbeard.EP_DEFAULT_DELETED_STATUS
+                                
                             logger.log(u"%s: Location for S%02dE%02d doesn't exist, removing it and changing our status to %s" %
-                            (self.indexerid, season, episode, statusStrings[sickbeard.EP_DEFAULT_DELETED_STATUS]) ,logger.DEBUG)
+                            (self.indexerid, season, episode, statusStrings[new_status]) ,logger.DEBUG)
                             curEp.status = sickbeard.EP_DEFAULT_DELETED_STATUS
```
Comment (miigotu) [inline sickbeard/tv.py]: `curEp.status = new_status`

Comment (fernandog) [inline sickbeard/tv.py]: ops. need to remove that, isn't? I already set that in the IF

**Comment by miigotu** [comment]
You didnt change the status, just the log line.
Add `ARCHIVED` to config_general.mako line 620.

**Comment by fernandog** [comment]
see line 1107 and 1109 in diff

**Comment by miigotu** [comment]
And then you change it right back to default ep status. you need to remove the line right after the log line.

File: sickbeard/tv.py (hunk 1 above)
Comment (miigotu) [inline sickbeard/tv.py]: Yes just remove it.

**Comment by fernandog** [comment]
but we need the new_status to log it

**Comment by miigotu** [comment]
so why cant you figure out new status, log it, and then set it.

**Comment by fernandog** [comment]
like this?

```

                            if sickbeard.EP_DEFAULT_DELETED_STATUS == 'ARCHIVED':
                                oldStatus, oldQuality = Quality.splitCompositeStatus(curEp.status)
                                new_status = Quality.compositeStatus(ARCHIVED, oldQuality)
                            else:
                                new_status = sickbeard.EP_DEFAULT_DELETED_STATUS
                                
                             logger.log(u"%s: Location for S%02dE%02d doesn't exist, removing it and changing our status to %s" %
                             curEp.status = new_status

```

**Comment by fernandog** [comment]
@miigotu feel free to merge if is ok to you

File: sickbeard/tv.py [hunk 2]
```diff
@@ -1100,9 +1100,16 @@ def refreshDir(self):
                     with curEp.lock:
                         # if it used to have a file associated with it and it doesn't anymore then set it to sickbeard.EP_DEFAULT_DELETED_STATUS
                         if curEp.location and curEp.status in Quality.DOWNLOADED:
+                        
+                            if sickbeard.EP_DEFAULT_DELETED_STATUS == 'ARCHIVED':
```
Comment (fernandog) [inline sickbeard/tv.py]: config is a integer:. In my case is 6

so here should be == ARQUIVED without quotes, isn't?

**Comment by miigotu** [comment]
yes, but lemme see it from my PC when I get home before merge.

**Comment by miigotu** [comment]
Good, just squash and make sure its still good and it can merge.

**Comment by fernandog** [comment]
@miigotu added better explanation:

image

and remove 'Skipped remove setting from PP' (not PP setting) and moved to 'general'. 
This settting enables/disables the "default deleted episode stauts". They should be together

**Comment by miigotu** [comment]
Ok, squash?

**Comment by fernandog** [comment]
thanks

**Comment by fernandog** [comment]
@OmgImAlexis  there's a issue in mako I can't fix. can you help?

That 6 is the sickbeard.EP_DEFAULT_DELETED_STATUS I added in .mako for this test.
See that none is 'selected'

```
                                    <span class="component-desc">
6
                                        <select name="ep_default_deleted_status" id="ep_default_deleted_status" class="form-control input-sm">
                                            <option value="5" >Skipped</option>
                                            <option value="7" >Ignored</option>
                                            <option value="6" >Archived</option>
                                        </select>
```
https://github.com/SiCKRAGETV/SickRage/blob/develop/gui/slick/views/config_general.mako#L634

**Comment by miigotu** [comment]
try `[int(sickbeard.EP_DEFAULT_DELETED_STATUS) == defStatus]`

**Comment by fernandog** [comment]
worked! I tried int in both but didnt worked. thanks

**Comment by fernandog** [comment]
https://github.com/SiCKRAGETV/SickRage/commit/fb960550e4a9c860be5ba830203e619e91ea9c32

**Comment by fernandog** [comment]
https://github.com/SiCKRAGETV/sickrage-issues/issues/510

**Pull request merged**